tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
Hardcoding paths here is a very bad idea since i'll not be able to run this on my machine since this path doesn't exist for me, right? 

Tests aren't only for Travis, they are to be run by every developer.